device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -749,6 +749,7 @@ public void subscribeToDeviceMethod(DeviceMethodCallback deviceMethodCallback, O
     }
 
     // Used by multiplexing clients to signal to this client what kind of multiplexing client is using this device client
+    @SuppressWarnings("SameParameterValue") // This can be the current multiplexing client or the deprecated transport client.
```

Comment:
can you explain what's the warning here about?

Reply:
the warning here complains that connection type is currently only set to "multiplexingClient" but I remember Tim mentioning that this can take in other values as well.
@timtay-microsoft to confirm.

Reply:
Nit: What you said is perfect for the comment here.